DOCPLAN-238: Fixing Vale errors (#111215)
openshift/openshift-docs: virt/about_virt/virt-security-policies.adoc

toc::[]

[role="_abstract"]

> {VirtProductName} provides built-in security features and authorization policies to protect `virtual machine` workloads and ensure secure cluster operations across your environment.

Reviewer: ```suggestion
{VirtProductName} provides built-in security features and authorization policies to protect virtual machine workloads and ensure secure cluster operations across your environment.
```
This formatting for VMs seems odd to me

*Key points*

* {VirtProductName} adheres to the `restricted` link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Kubernetes pod security standards] profile, which aims to enforce the current best practices for pod security.
* Virtual machine (VM) workloads run as unprivileged pods.

> * Security context constraints (SCCs) are defined for the `kubevirt-controller` service account. For more information about SSCs, refer to the Additional Resourcses section.

Reviewer: ```suggestion
* Security context constraints (SCCs) are defined for the `kubevirt-controller` service account. For more information about SSCs, see "Additional resources".
```
Fix typo and also formatting per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

[id="authorization_virt-security-policies"]
== Authorization

> {VirtProductName} uses Role-based access control(RBAC) to define permissions for human users and service accounts. The permissions defined for service accounts control the actions that {VirtProductName} components can perform.

Reviewer: ```suggestion
{VirtProductName} uses role-based access control (RBAC) to define permissions for human users and service accounts. The permissions defined for service accounts control the actions that {VirtProductName} components can perform.
```

[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources

> * xref:../../authentication/managing-security-context-constraints.adoc#security-context-constraints-about_configuring-internal-oauth[Security context constraints] 

Reviewer: ```suggestion
* xref:../../authentication/managing-security-context-constraints.adoc#security-context-constraints-about_configuring-internal-oauth[About security context constraints] 
```
Same comment - check the link text against the section it's linking to for all xrefs

> * xref:../../authentication/using-rbac.adoc#using-rbac[role-based access control]

Reviewer: ```suggestion
* xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions]
```
Link text should always be the title it's linking to - please check the names for all links
